src/stream/src/common/table/state_table.rs

```diff
@@ -938,6 +934,21 @@ where
             .map(get_second))
     }
 
+    pub async fn iter_all_vnodes_with_pk_range(
```

Comment:
New interface added for state_table, for iterating pk_range for ALL vnodes of the state table.